### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by george-zakharov**
Hi!

I've made some minor style fixes.
Regards

### CODE REVIEW

File: autoload.php [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <?php
 
-require __DIR__.'/src/functions_include.php';
+require __DIR__ . '/src/functions_include.php';
 
-spl_autoload_register(function ($class) {
+spl_autoload_register(static function ($class) {
```
Comment (nyamsprod) [inline autoload.php:5]: the addition of `static` is more of a micro-optimization than anything else I would not add it here

File: src/AbstractCsv.php [hunk 2]
```diff
@@ -323,7 +323,7 @@ public function output(string $filename = null): int
 
         echo $this->output_bom;
 
-        return strlen($this->output_bom) + (int) $this->document->fpassthru();
+        return strlen($this->output_bom) + (int)$this->document->fpassthru();
```
Comment (nyamsprod) [inline src/AbstractCsv.php:326]: Why removing the space ??

File: src/CharsetConverter.php [hunk 3]
```diff
@@ -94,7 +94,7 @@ protected static function filterEncoding(string $encoding): string
             return $encoding_list[$key];
         }
 
-        throw new OutOfRangeException('The submitted charset '.$encoding.' is not supported by the mbstring extension.');
+        throw new OutOfRangeException('The submitted charset ' . $encoding . ' is not supported by the mbstring extension.');
```
Comment (nyamsprod) [inline src/CharsetConverter.php:97]: Why adding spaces ? Did you run `composer phpcs` ?

File: src/TabularDataReader.php [hunk 4]
```diff
@@ -48,7 +48,7 @@ public function getIterator(): Iterator;
     /**
      * Returns the header associated with the tabular data.
      *
-     * The header must contains unique string or is an empty array
+     * The header must contain unique string or to be an empty array
```
Comment (nyamsprod) [inline src/TabularDataReader.php:51]: good catch

**Comment by nyamsprod** [comment]
@george-zakharov I like the typo fixes but I am a bit against the coding style changes you added in the PR. Those coding style exists since v9 and even if I want to change some I will probably update them for v10 whenever that one comes alive but not before to stay as consistent as possible.
If you could update your PR and only submit the typo changes I would gladly merge them ASAP

File: autoload.php (hunk 1 above)
Comment (george-zakharov) [reply autoload.php:5]: Okay, I see.

File: src/AbstractCsv.php (hunk 2 above)
Comment (george-zakharov) [reply src/AbstractCsv.php:326]: Well, it's an interesting thing :)
I did it according to my habbits. I've checked PSR once again and haven't find the any clarity about such situation.
I think that [here](https://github.com/php-fig-rectified/fig-rectified-standards/issues/13#issuecomment-324653158) is a pretty good explanation why. I agree with it.
Anyway, according to the inner code style of this project my fix is not neccessary.

File: src/CharsetConverter.php (hunk 3 above)
Comment (george-zakharov) [reply src/CharsetConverter.php:97]: Well, I've checked standards and haven't find there any consistency about spaces around concatenation dot.
My point is readability matters.
Anyway, I think we don't need my fix here :)

**Comment by george-zakharov** [comment]
> 

Okay, I'll fix it :)

**Comment by george-zakharov** [comment]
@nyamsprod did it :)

**Comment by nyamsprod** [comment]
@george-zakharov thanks. By the way now that I saw your answer I get it. League CSV uses PSR-2 as the basis of it's coding style + some additions by me. It was done prior to PSR-12 or PSR-2R was even created This explain more my why's 😉 . Anyway I will review the package coding style when the next major version is released. Thanks for the PR in any case.

**Comment by george-zakharov** [comment]
@nyamsprod thank you too!
I will try to contribute further :)

**Pull request merged**